src/main/kotlin/com/coxautodev/graphql/tools/MethodFieldResolver.kt

```diff
@@ -117,17 +116,7 @@ internal class MethodFieldResolver(field: FieldDefinition, search: FieldResolver
         return if (batched) {
             BatchedMethodFieldResolverDataFetcher(getSourceResolver(), this.method, args, options)
         } else {
-            if (args.size == 0
-                    && this.method.parameterCount == 0
-                    && this.method.name.startsWith("get")
-                    && this.search.type is java.lang.Class<*>
-                    && (this.search.type as java.lang.Class<*>).getMethod(this.method.getName()) != null
```

Comment:
@cliedeman This line throws a `NoSuchMethodException` for methods with package private scope therefore breaking backwards compatibility. Can you add unit tests to ensure it's working properly including the package private use case? Once all that's working I can merge it again.